Add test vectors for batched privately verifiable tokens and arbitrary (#31)
ietf-wg-privacypass/ietf-draft-privacypass-batched-tokens: draft-ietf-privacypass-batched-tokens.md

: IETF
{: spacing="compact"}

--- back

# Test Vectors

This section includes test vectors for the two issuance protocols
specified in this document. {{test-vectors-batched-voprf}} contains test vectors

> for batched token issuance protocol (0x0005), and {{test-vectors-arbitrary-batched}} contains

Reviewer: ```suggestion
for Amortized Privately Verifiable Token Batch Issuance (0x0005), and {{test-vectors-arbitrary-batched}} contains
```

> test vectors for arbitrary batched token issuance protocol.

Reviewer: ```suggestion
test vectors for Generic Token Batch Issuance.
```

> ## Issuance Protocol 1 - VOPRF (ristretto255, SHA-512) {#test-vectors-batched-voprf}

Reviewer: ```suggestion
## Amortized Privately Verifiable Token Batch Issuance - VOPRF (ristretto255, SHA-512) {#test-vectors-batched-voprf}
```

  ded70ebe501ef6b44db79b406dd1fc90e5313f93cdc37cbe1616fcfee47919d
  dbb18e79b1c5b59c73b117cca3266bd5cd062865c13f2d474b241dfd6b23321
  85d205a3327
  - 00057270ebf05813dfaebd76f29d4ee2402999a9acaf082ce462479fabbf2
  a70493f05bebf3881b04fbb652613bd335710fce992e473ae4c5d860066d5f1
  5535bcd1f1c2acd113354cf4232d21f040e81e072d5ffaa444cc624c262eb96
  ded70ebe5fcd83a3c259a5cfd9ab76ef64eca274475d0c74860ea549fa21257
  eaed4a22d467f19c7a90dd7840c2d377735f3b8acaafd35bdb2b2bf39cf030e
  4c412c32f29
  - 000508ac07ee5eee937f0f1a8965e8b0a18eb51466dacd1861bc6e6727c0e
  140b32f05bebf3881b04fbb652613bd335710fce992e473ae4c5d860066d5f1
  5535bcd1f1c2acd113354cf4232d21f040e81e072d5ffaa444cc624c262eb96
  ded70ebe55483b60d23c4def013fb9d349909d9bfa155d57df150c34472ca11
  a269e2fe47cd41e1254e5ed9f09afb7d1c3bcf5fcc256017b97947d25de766c
  4f0acd38362
  - 0005059bb7481ed568c5abcabb533404e464b381a4e4874c1e7bcb1058183
  f3018b905bebf3881b04fbb652613bd335710fce992e473ae4c5d860066d5f1
  5535bcd1f1c2acd113354cf4232d21f040e81e072d5ffaa444cc624c262eb96
  ded70ebe5a7d2b62940f03fd3d0c4a20a323c8758a4e3576c92d00156dc1f2d
  2ff9ebe900d5d2e6ecbb6a97cdbf8f185264264dd303558815d9308d61e567f
  3df9b274bbb
~~~

> ## Issuance Protocol 2 - Generic Token Batch {#test-vectors-arbitrary-batched}

Reviewer: ```suggestion
## Generic Token Batch Issuance {#test-vectors-arbitrary-batched}
```
